### ISSUE TITLE
Certain websockets-related items need accessibility text

### ISSUE THREAD

**Issue by dsesami** [ansible/awx#7677]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
Some components need to have text so that they can be targeted by screenreaders and automation. Two components that come to mind are the status square (seen in project syncs, jobs list status, and job templates list), as well as the cloud sync status (the cloud symbol on the inventory list). 

This ticket is for adding text to these elements, as well as ensuring that the state of the elements (success, failure, running, error) is reflected in the text.

**Comment by unlikelyzero** [comment]
@keithjgrant bonus points if the elements had some additional identifiers (job id, job name, etc) but may not be necessary

**Comment by tiagodread** [comment]
Verified and PR already merged!
